docs/store/datastore.md

```diff
@@ -3,7 +3,7 @@
 
 A data store defines a storage provider (e.g. file system, S3, Azure blob, Iguazio v3io, etc.).
 
-MLRun supports multiple data stores. ((Additional data stores, for example MongoDB, can easily be added by extending the `DataStore` class.)
+MLRun supports multiple data stores. (Additional data stores, for example MongoDB, can easily be added by extending the `DataStore` class.)
```

Comment:
Seems to me like the parentheses are unnecessary, and arguably make for incorrect punctuation.
```suggestion
MLRun supports multiple data stores. Additional data stores, for example MongoDB, can easily be added by extending the `DataStore` class.
```